doc/FREQUENTLY_MADE_SUGGESTIONS.md

```diff
@@ -283,6 +283,24 @@ Storing blood for later is far outside the reach of the survivor even if they kn
 
 Coercing followers into being your personal blood bags is not going to happen. Otherwise, this would require someone with medical knowhow, matching blood types, about an hour of sitting around for the transfusion, and would make the recipient violently ill for about a day. But there is a better solution.
 
+#### Add 3D printers and have them print guns and armor and car parts: Yes but no.
+
+This breaks down into two possibilities, which tl;dr, neither really works.
+The first option is you find design files for 3D prints. This is extremely problematic for a number of reasons.  There's no internet, we have no robust way to search for files on people's computers, and many of the designs of particular interest are low-grade contraband, so where are you getting them? "finding files on USB sticks in the trash" is already stretched past its breaking point, it is most definitely not reasonable to further extend it to cover an entirely new way of crafting items.
+
+Even if you find a house or workplace with 3D printers (which is a safe assumption, because otherwise you aren't going to have a 3D printer in the first place), most of the design files you're going to find are going to be figurines, toys, and replacement parts for random appliances, not things you waThere are a number of issues with adding 3D printing. One, your assertion that ability to 3D print is based on the user's ability to operate a 3D printer assumes that they are working not just from a design, but an extremely well proven design. In reality there are many designs out there you need to heavily modify in order to get it printing, at which point you have to repeat the final stages of testing and adjusting the design in order to get a working output. This is much more the case for the survivor scenario because you're interested in working solutions instead of things that "look right".
```

Comment:
> things you waThere are 

Merge error?

Reply:
"nt/need" after "...just isn't that tough of a material" connects nicely to follow "wa". perhaps missed the click by a couple characters before copy-pasting his next reply